test/object-store/realm.cpp

```diff
@@ -1079,6 +1080,21 @@ TEST_CASE("SharedRealm: convert") {
         REQUIRE(sync_realm->read_group().get_table("class_object")->size() == 1);
     }
 
+    SECTION("cannot convert a local flx sync") {
```

Comment:
This is missing some words or something.

Reply:
fixed